lrathoni/Lullaby-for-a-fish
# Fixed Standalone build

### What

* Fixed Standalone builds for Windows and Linux.
* Fixed submodule inclusion of MptSharp.

### How to test

* Checkout this branch.
* Build Unity project on Windows or Linux 64-bit (development build).
* Launch executable.
* Check that there are no library import errors.
* Check that sound plays correctly (same as checks on previous Pull Request).
